tekton/README.md

```diff
@@ -1,60 +1,61 @@
 # Tekton Dashboard CI/CD
 
-This directory contains the Tekton `Tasks` and `Pipelines` used to create Dashboard releases. 
+This directory contains the Tekton `Tasks` and `Pipelines` used to create Dashboard releases.
 
-These tasks run on your local cluster, and then copy the release artifacts - docker images and yaml files - into [the `tekton releases` bucket in Google Cloud Platform](https://console.cloud.google.com/storage/browser/tekton-releases/dashboard). Your cluster much contain keys from rom a google account with the necessary authorization in order for this to work. 
+These tasks run on your local cluster, and then copy the release artifacts - docker images and yaml files - into [the `tekton releases` bucket in Google Cloud Platform](https://console.cloud.google.com/storage/browser/tekton-releases/dashboard). Your cluster much contain keys from rom a google account with the necessary authorization in order for this to work.
```

Comment:
May as well fix up `from rom` while you're here

Reply:
done